docs/AboutUs.md

```diff
@@ -7,17 +7,14 @@ We are a team based in the [School of Computing, National University of Singapor
 
 ## Project team
 
-### John Doe
+### Yu Zizhen
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/yuzizhen.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](https://github.com/YuZizhen)]
 
-* Role: Project Advisor
-
-### Koh Guan Zeh
```

Comment:
I think you accidentally deleted this line

Reply:
AHHH ok i have added it back SORRY